pkg/autoscaler/statforwarder/processor.go

```diff
@@ -54,11 +74,49 @@ func (p *bucketProcessor) process(sm asmetrics.StatMessage) error {
 		return err
 	}
 
-	return p.conn.SendRaw(gorillawebsocket.BinaryMessage, b)
+	if p.podAddressable && p.podConn.SendRaw(gorillawebsocket.BinaryMessage, b) == nil {
+		// Pod is accessible via IP address, close the connection via SVC.
+		if p.svcConn != nil {
```

Comment:
I don't think we can end up here, since initially the conn is `nil` and once `podAddressable` is set to `false` it can't be unset.